templates/terraform/nested_property_documentation.erb

```diff
@@ -1,12 +1,12 @@
 <%
-  nested_properties = effective_nested_properties(config, property)
+  nested_properties = effective_nested_properties(property)
   if !nested_properties.empty?
 -%>
 The `<%= Google::StringUtils.underscore(property.name) -%>` block <%= if property.output then "contains" else "supports" end -%>:
 <% nested_properties.each do |prop| -%>
-<%= build_property_documentation(config, prop) -%>
+<%= build_property_documentation(prop) -%>
```

Comment:
Idea for next feature ;) ... Enable adding functions to Api::Object resources and properties. That way we could put Provider on a diet and move code like this to the property (or resource) directly:

```
<%= prop.documentation -%>
```

Reply:
This is one of the nice feature this model can enable :) Totally agree.